upgrading/manual_upgrades.adoc

```diff
@@ -507,80 +507,42 @@ updated.
 [[updating-policy-definitions]]
 == Updating Policy Definitions
 
-After a cluster upgrade, the default roles
+During a cluster upgrade, and on every restart of any master, the 
 xref:../architecture/additional_concepts/authorization.adoc#roles[default
-cluster roles] are automatically updated. To check if all defaults are set as
-recommended for your environment, run:
+cluster roles] are automatically updated. 
 
+. If you customized default cluster roles and want to ensure a role reconciliation
+does not modify them, protect each role from reconciliation:
++
```

Comment:
@kalexand-rh Do we need to define _reconciliation_ here?  Not sure we define it anywhere.